service.kn.switchtimer/addtimer.py

```diff
@@ -0,0 +1,17 @@
+import xbmc
+import handler
+
+if __name__ ==  '__main__':
+
+    handler.notifyLog('Parameter handler called: add Timer')
+    try:
+        args = {'channel':xbmc.getInfoLabel('ListItem.ChannelName'),
```

Comment:
This is the only way I know to get infos from within a PVROSDGuide window. If you look inside a skin you'll find only those named ListItems.xxx.